lib/galaxy/webapps/galaxy/controllers/workflow.py

```diff
@@ -646,13 +646,20 @@ def delete(self, trans, id=None):
 
     @web.expose
     @web.require_login("edit workflows")
-    def editor(self, trans, id=None, version=None):
+    def editor(self, trans, id=None, workflow_id=None, version=None):
         """
         Render the main workflow editor interface. The canvas is embedded as
         an iframe (necessary for scrolling to work properly), which is
         rendered by `editor_canvas`.
         """
         if not id:
+            if workflow_id:
+                workflow = trans.sa_session.query(model.Workflow).get(trans.security.decode_id(workflow_id))
+                stored_workflow = workflow.stored_workflow
+                self.security_check(trans, stored_workflow, True, False)
+                stored_workflow_id = trans.security.encode_id(stored_workflow.id)
+                return trans.response.send_redirect(f'{url_for("/")}workflow/editor?id={stored_workflow_id}')
```

Comment:
Yeah - the GUI says "load the subworkflow at the latest version" so that is what is happening here.

Loading an older version in the editor would be bad because you might save over more recent changes. I think having a read-only view of the workflow from the editor though would be awesome - so users can see what is happening in the subworkflow but that wasn't the intention here.